src/System.Resources.ResourceManager/ref/project.json

```diff
@@ -1,14 +1,11 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Reflection": "4.0.0",
-    "System.Globalization": "4.0.0"
+    "System.IO.UnmanagedMemoryStream": "4.3.0-beta-24604-02",
```

Comment:
That should work.  Looks like InteropServices did not depend on ResourceManager before.